modules/channel-web/src/views/lite/components/messages/renderer/FileMessage.tsx

```diff
@@ -13,7 +12,16 @@ export const FileMessage = (props: Renderer.FileMessage) => {
 
   const { url, title, storage, text } = props.file
 
-  const extension = path.extname(url)
+  let extension = ''
+  try {
+    const ValidUrl = new URL(url)
+
+    extension = ValidUrl.pathname
+  } catch (error) {
+    // If the URL is not valid return a dummy component.
+    return null
```

Comment:
What's the outcome if you happen to have an invalid URL? Does it display nothing or a bubble with an error message?

Reply:
I will try !